share/messages_to_error_log.txt

```diff
@@ -12353,39 +12346,13 @@ ER_GRP_RPL_MYSQL_NETWORK_PROVIDER_SERVER_ERROR_COMMAND_ERR
 # See the FAQ in errmsg_readme.txt in the
 # same directory as this file for more
 # information.
-<<<<<<< HEAD
-||||||| b5da0b9817c
 
-=======
 
-
->>>>>>> mysql-8.1.0
 #
 # End of 8.1 error messages intended to be written to the server error log.
 #
 
-<<<<<<< HEAD
-#
-# Start of Percona Server 8.0 server error log messages
-#
-start-error-number 48000
-
-ER_KEYRING_COMPONENT_NO_CONFIG
-  eng "Keyring configuration doesn't exists: %s"
-
-ER_KEYRING_COMPONENT_CONFIG_PARSE_FAILED
-  eng "Keyring configuration JSON parse error: %s (%lu)"
-
-ER_KEYRING_COMPONENT_KMS_ERROR
-  eng "KMS communication error: %s"
-
-#
-# End of Percona Server 8.0 server error log messages
```

Comment:
PXB needs these error messages.

Reply:
They are only moved to the end. Upstream added a section for 8.1 messages that numerically (start-error-number) must come right after the original 8.0 error codes because their start error number is lower the Percona ones.

https://github.com/altmannmarcelo/percona-xtrabackup/blob/PXB-3105-8.1/share/messages_to_error_log.txt#L12358 - is were they are located now

Reply:
Right, I realized that in the next commit they are present at end